src/modules/scores.js

```diff
@@ -1,2 +1,32 @@
-const scores = [100, 20, 50, 78, 125, 77, 42];
-export default scores;
+import API_URL from './config.js';
+
+class Scores {
+  static scores = [];
+
+  static name = ''
+
+  static requestOptions = {
+    method: 'POST',
+    headers: { 'Content-Type': 'application/json' },
+  };
+
+  static AddNewScore = async (score) => {
+    this.requestOptions.body = JSON.stringify(score);
+    await fetch(`${API_URL}games/${this.name}/scores`, this.requestOptions);
+    this.scores.push(score);
+  };
+
+  static createGameName = async (title = 'veronica365-1') => {
+    this.requestOptions.body = JSON.stringify({ name: title });
+    const response = await fetch(`${API_URL}games/`, this.requestOptions);
+    const { result } = await response.json();
+    const name = String(result).split(' ')[3];
+    this.name = name;
+  };
+
+  static getAllScores = async () => {
+    const response = await fetch(`${API_URL}games/${this.name}/scores`);
+    this.scores = (await response.json()).result;
+  };
```

Comment:
- [ ] Good job as the add and refresh button is working fine, but when a user initially lands on the page or refreshes the browser, the scores do not appear. Please ensure that when a user refreshes the browser or initially lands on the page, it fetches the scores and displays it.